src/services/FDC3/desktopAgentClient.ts

```diff
@@ -2,11 +2,12 @@ import Channel from "./channelClient";
 import { EventEmitter } from "events";
```

Comment:
And this file - revert it if not significant changes. Also check your editor's formatting setup
Company ploicy, last I heard, was tabs - although linter appears setup for spaces... SO I don't actually know what we're supposed to use right now